.github/workflows/prodsec-workflow.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
     - main
     - develop
+    - cspl_2165_fix_sa_bugs
```

Comment:
Probably dont need this.

Reply:
Yes will be removing this after review approvals.